packages/playground/remote/src/lib/worker-thread.ts

```diff
@@ -275,6 +274,42 @@ try {
 		);
 	}
 
+	const wpStaticAssetsDir = wpVersionToStaticAssetsDirectory(
+		apiEndpoint.loadedWordPressVersion
+	);
+	const remoteAssetListPath = joinPaths(
+		requestHandler.documentRoot,
+		'wordpress-remote-asset-paths'
+	);
+	// Use a common WP static asset to guess whether we are using a minified WP build.
+	// This file is present in WP 6.2 - 6.5, nightly, and the current beta.
+	const commonStaticAssetPath = joinPaths(
+		requestHandler.documentRoot,
+		'wp-admin/css/common.css'
+	);
+
+	if (
+		wpStaticAssetsDir !== undefined &&
+		!primaryPhp.fileExists(remoteAssetListPath) &&
```

Comment:
Shouldn't this check if the remoteAssetListPath _exists_ (vs _doesn't exist_)? 

```suggestion
		primaryPhp.fileExists(remoteAssetListPath) &&
```

Reply:
Oh, I see, you're checking if that file is missing from the WordPress build and if we need to fetch it. Hm. I'm not sure I follow – I thought that list is only supposed to exist in the minified builds? Official WordPress releases don't ship one.

Reply:
I think it is correct as-is because we want to retrieve the file on-demand if it does not already exist.

Reply:
What would be a case where it doesn't exist in a minified zip bundle but does exist on the server?

Reply:
Aha, that would be the case for builds that are already cached in the browser storage today.

Reply:
> Oh, I see, you're checking if that file is missing from the WordPress build and if we need to fetch it. Hm. I'm not sure I follow – I thought that list is only supposed to exist in the minified builds? Official WordPress releases don't ship one.

@adamziel That's the challenge here. We don't want to retrieve this listing unless it is needed. It normally just comes with minified WP builds, but once we start depending on the existence of the listing file (#1532) and encounter an older minified WP build in browser or device storage, we need to retrieve the listing so the minified build continues to work. That was also the motivation of checking for the single `wp-admin/css/common.css` file as a hint that we are potentially dealing with a minified build.

Do you have another idea about how we could or should approach this so we don't retrieve anything unnecessarily or add to the filesystem of full WP builds? Note this is the worker-thread in `@wp-playground/remote`, in case that influences your thinking.

cc @bgrgicak because I think the question of detecting full WP also affects #1532.

Reply:
What is it with our timing here? 😄   I keep responding just as you respond in a way that makes my comment stale. 😂 

Thanks for looking at this again today!

Reply:
> Aha, that would be the case for builds that are already cached in the browser storage today.

This is actually why I reverted the routing changes the second time... because of existing browser storage without the file.